Makefile

```diff
@@ -122,6 +124,10 @@ else
 	@exit 1
 endif
 
+golangci-lint:
```

Comment:
I'm not sure we can directly use downloaded content here ,
@MarSik  WDYT?

Reply:
we are doing something similar with ginkgo for other tests ...

Reply:
@yanirq @MarSik 
kindly reminder

Reply:
Downloading is fine as long as the downloading does not happen in the (ART) environment production (nightly/release) images are built.